inventory_management/inventory_management/doctype/stock_entry/stock_entry.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2024, Viny Selopal and contributors
+// For license information, please see license.txt
+
+// frappe.ui.form.on("Stock Entry", {
+// 	refresh(frm) {
```

Comment:
I just noticed the client-side controller is still missing

1. Add a custom button to view filtered Stock Ledger report related to this transaction. Check this comment in `stock_ ledger_entry.json`
2. If stock entry type = "Consume", you will have to fetch the current valuation rate from the last SLE (API call) and set it in the row for that item & warehouse. We are maintaining ledgers to automate this precisely. The Rate field will be read-only for Consume.